packages/Uploader/stories/Testing.js

```diff
@@ -0,0 +1,146 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { css } from "styled-components";
+import tokens from "@paprika/tokens";
+import Uploader, { UploaderContext } from "../src/Uploader";
+
+const styles = {
+  table: css`
+    td {
+      max-width: 210px;
+      overflow: hidden;
+      padding: 8px;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+    }
+  `,
+  fileInput: css`
+    align-items: center;
+    border: 1px dashed blue;
+    border-radius: 6px;
+    box-shadow: ${tokens.popover.shadow};
+    display: flex;
+    height: 200px;
+    justify-content: center;
+    width: 300px;
+  `,
+};
+const propTypes = {
+  hasUploadButton: PropTypes.bool,
+};
+
+const defaultProps = {
+  hasUploadButton: false,
+};
+
+function hasError(isValid) {
+  return isValid ? "👍" : "🚨";
+}
+
+export default function Testing(props) {
+  const {
+    /* custom prop from storybook just fo this example */
+    hasUploadButton,
+  } = props;
+
+  const {
+    files,
+    isDraggingOver,
+    isDragLeave,
+    isDisabled,
+    isCompleted,
+    upload,
+    removeFile,
+    cancelFile,
+    FileInput,
+  } = React.useContext(UploaderContext);
+
+  const handleUpload = () => {
+    upload();
+  };
+
+  const handleRemove = key => () => {
+    removeFile(key);
+  };
+
+  const handleCancel = key => () => {
+    cancelFile(key);
+  };
+
+  return (
+    <>
+      <FileInput>
+        <div css={styles.fileInput}>FILE INPUT AREA (children)</div>
+      </FileInput>
+      {hasUploadButton ? (
+        <p>
+          After selecting your images, click the button to upload it.
+          <button type="button" onClick={handleUpload}>
+            upload images
+          </button>
+        </p>
+      ) : null}
+      <p>{isDisabled ? "isDisabled is true" : "isDisabled is false"}</p>
+      <p>{isDraggingOver ? "isDraggingOver 🤚" : ""}</p>
+      <p>{isDragLeave ? "isDragLeave 👋" : ""}</p>
```

Comment:
Apart from logging this event here, none of the stories illustrate how (or why) this prop is used.  I'm still not convinced it's necessary – I think `!isDraggingOver` should suffice?